pilot/pkg/networking/util/util.go

```diff
@@ -75,11 +75,11 @@ const (
 
 	// EnvoyRawBufferSocketName matched with hardcoded built-in Envoy transport name which determines
 	// endpoint level plantext transport socket configuration
-	EnvoyRawBufferSocketName = "raw_buffer"
+	EnvoyRawBufferSocketName = "envoy.transport_sockets.raw_buffer"
```

Comment:
@lizan just to confirm, envoy supporting matching on the both naming, and changing this name is backward compatible?

Reply:
As far as I know, they are equivalent except one makes the logs spam 100s of times that its deprecated and one does not :slightly_smiling_face: 

The only issue may be if its a recent addition to Envoy, should confirm that

Reply:
can you add some code link of this to confirm, just for reference sake?